tests/codebuild/create_tests.sh

```diff
@@ -107,7 +111,10 @@ function verify_test()
   fi
 
   echo "Waiting for job to complete"
-  wait_for_crd_status_else_fail "$crd_type" "$crd_instance" "$timeout" "$desired_status"
+  if ! wait_for_crd_status_else_fail "$crd_type" "$crd_instance" "$timeout" "$desired_status"; then
+      echo "[FAILED] Waiting for status Completed failed"
```

Comment:
Can we remove the log from the function so that this message is not duplicated? It will make our usage of `wait_for_crd_status_else_fail` more consistent, since printing the failure message is owned by the caller.